### ISSUE TITLE
NodeSshKeyAdd and NodeSshKeyRemove do not consider offline nodes properly

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1103]
Originally reported of Google Code with ID 1047.

```
The online/offline information would need to be propagated in the RPC calls.
```
Originally added on 2015-03-20 13:25:16 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
Fixed with:

commit 49f49f81148ffcde660eee58ccfdafb2010e2329
Author: Helga Velroyen <helgav@google.com>
Date:   Thu Apr 16 15:02:47 2015 +0200

    Consider offline nodes when removing SSH keys
    
    Like the previous patch, also NodeSshRemoveKey did not
    consider offline nodes so far. This patch fixes it.
    
    This fixes issue 1047.
    
    Signed-off-by: Helga Velroyen <helgav@google.com>
    Reviewed-by: Petr Pudlak <pudlak@google.com>

commit d95ba4afd730c87198cfd17ae32e5211a2723d64
Author: Helga Velroyen <helgav@google.com>
Date:   Thu Apr 16 15:02:08 2015 +0200

    Consider offline nodes in NodeSshKeyAdd
    
    So far the backend function NodeSshKeyAdd naively assumed
    that all nodes are online and reachable via SSH. This
    patch adds offline status information to the function's
    signature to respect the offline status when updating
    SSH keys.
    
    Signed-off-by: Helga Velroyen <helgav@google.com>
    Reviewed-by: Petr Pudlak <pudlak@google.com>
```
Originally added on 2015-04-23 15:00:34 +0000 UTC.

Mentioned Issue with GoogleCode ID 1047 has been migrated to GitHub with ID #1103.
Changed State: Fixed

**Comment by Ganeti-Issues-Migrator** [comment]
```
Released as part of 2.14.0
```
Originally added on 2015-06-02 10:37:03 +0000 UTC.

Changed State: Released
